### PR TITLE
Adding / modifying content for Distribution type

### PR DESCRIPTION
**Pull request by phrawzty**
### What does this PR do?

Adds more detail around the new Distribution type, specifically as regards the Developer Tools section.

### Motivation

Making the new Distribution type easier to find and understand.

### Preview link

* https://docs-staging.datadoghq.com/phrawzty/distribution_is_a_thing/developers/metrics
* https://docs-staging.datadoghq.com/phrawzty/distribution_is_a_thing/developers/metrics/distributions/
* https://docs-staging.datadoghq.com/phrawzty/distribution_is_a_thing/developers/dogstatsd/data_types/#distributions
* https://docs-staging.datadoghq.com/phrawzty/distribution_is_a_thing/graphing/metrics/distributions/

### CODE REVIEW

File: content/developers/metrics/distributions.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Distributions
+kind: documentation
+further_reading:
```
Comment (l0k0ms) [inline content/developers/metrics/distributions.md]: We should add a link to https://docs.datadoghq.com/graphing/metrics/distributions/ on this page to encourage them to learn more about the dedicated UI for distribution metrics.

**Comment by mikezvi** [comment]
Graphing > Metrics > distributions seems very uncoupled from the others, but also doesn't have a lot to do with graphing.

I think we should link to here from "Data Types and Tags" under "Tagging", if nothing else.  Otherwise the refactor is much cleaner.

**Comment by mikezvi** [comment]
Now that we've split things out a bit, should we provide a toy example of why Distributions are so important, or is it clear in the phrasing `Because averages of percentile aggregations are not statistically valid...`

Extremely simple example I've been using:

Say host foo reports a metric with the values [1,1,1,2,2,2,3,3] and host bar reports the same metric with the values [1,1,2] during a flush interval.   

Here, the p50 (median) for foo is 2 and the p50 for bar is 1.  Aggregating by the average value would result in 1.5.  

In reality, the global p50 (median) is the median of the combined set, [1,1,1,1,1,2,2,2,2,3,3], which is 2.

File: content/graphing/metrics/distributions.md [hunk 2]
```diff
@@ -30,6 +34,10 @@ With the [distribution UI][3], create additional aggregate timeseries by applyin
 
 {{< img src="graphing/metrics/distributions/distribution_metric.png" alt="Distribution metric" responsive="true" >}}
 
+When creating your own graph, distribution metrics automatically have additional space aggregations available in the UI:
```
Comment (cswatt) [inline content/graphing/metrics/distributions.md]: capitalize Distribution for consistency across the doc

**Comment by phrawzty** [comment]
@mikezvi 
> I think we should link to here from "Data Types and Tags" under "Tagging", if nothing else. Otherwise the refactor is much cleaner.

Could you please provide full nav paths or URLs? I'm not sure what sections you're referring to here.

**Comment by phrawzty** [comment]
@mikezvi 
> Extremely simple example I've been using: […]

That's a great example and I have 100% nicked it for the documentation. 😄

**Comment by mikezvi** [comment]
Sorry!  There is "Tagging" documentation here: https://docs-staging.datadoghq.com/phrawzty/distribution_is_a_thing/developers/dogstatsd/data_types/#tagging

This page includes the Distribution type, but tagging for Distributions is not as simple as this model.  I don't think we need to put a ton of words explaining that right here, but we should have a simple "See advanced tagging operations for Distributions here: https://docs-staging.datadoghq.com/phrawzty/distribution_is_a_thing/graphing/metrics/distributions/" or something that reads better.

**Pull request merged**